src/main/java/org/sonarlint/intellij/actions/OpenSecurityHotspotInDialogAction.kt

```diff
@@ -0,0 +1,19 @@
+package org.sonarlint.intellij.actions
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.Project
+
+class OpenSecurityHotspotInDialogAction : AbstractSonarAction(
+    "Open In Dialog",
+    "Open security hotspot in dialog",
+    null
+) {
+    override fun updatePresentation(e: AnActionEvent, project: Project) {
```

Comment:
I don't think you have to override `updatePresentation`, this is mostly used when text is dynamic (having to adjust title on SQ or SC for example)